ow_lander/launch/spawn.launch

```diff
@@ -100,7 +100,8 @@
     output="screen"/>
   <node pkg="ow_lander" name="AntennaPanTiltAction"
     type="antenna_pan_tilt_action_server.py"
-    launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
+    launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "
+    output="screen"/>

```

Comment:
Good addition, it may explain why I wasn't seeing INFO messages in my current antenna work.